packages/apt/test.sh

```diff
@@ -12,11 +12,20 @@ echo "debconf debconf/frontend select Noninteractive" | \
 sudo apt update
 sudo apt install -V -y apparmor lsb-release wget
 
+distribution=$(lsb_release --id --short)
+case ${distribution} in
+  Debian)
+    repository=main
+    ;;
+  Ubuntu)
+    repository=universe
+  ;;
+esac
 code_name=$(lsb_release --codename --short)
 architecture=$(dpkg --print-architecture)
 
 wget \
-  https://packages.groonga.org/debian/groonga-apt-source-latest-${code_name}.deb
+  https://packages.groonga.org/${distribution,}/groonga-apt-source-latest-${code_name}.deb
```

Comment:
```suggestion
  https://packages.groonga.org/${distribution}/groonga-apt-source-latest-${code_name}.deb
```

Reply:
`lsb_release --id --short` returns a string that is the upper case for only the first character (e.g. `Ubuntu`, `Debian`.)
Therefore, we can't use it in the URL as it is now. 
I change `$(lsb_release --id --short)` to the lower case with `{distribution,}`.

Reply:
Oh, I didn't know `${XXX,}` feature!
But it's a bit tricky... How about converting `lsb_release --id --short` output?

```bash
distribution=$(lsb_release --id --short | tr 'A-Z' 'a-z')
```